win64.mak

```diff
@@ -124,9 +124,11 @@ custom_gc:
 test_shared:
 	$(MAKE) -f test\shared\win64.mak "DMD=$(DMD)" MODEL=$(MODEL) "VCDIR=$(VCDIR)" DRUNTIMELIB=$(DRUNTIME) "CC=$(CC)" test
 
+test_thread:
+	$(MAKE) -f test\thread\Mkefile "DMD=$(DMD)" MODEL=$(MODEL) "VCDIR=$(VCDIR)" DRUNTIMELIB=$(DRUNTIME) "CC=$(CC)" all
```

Comment:
s/Mkefile/Makefile/, but windows still builds with DM make, so a separate makefile is needed in the test directory.